src/WorkOS.net/Services/MFA/_interfaces/EnrollTotpFactorOptions.cs

```diff
@@ -0,0 +1,25 @@
+namespace WorkOS
+{
+    using Newtonsoft.Json;
+
+    public sealed class EnrollTotpFactorOptions : EnrollFactorOptions
+    {
+        public EnrollTotpFactorOptions(string issuer, string user)
+        {
+            this.Issuer = issuer;
+            this.User = user;
+        }
+
+        /// <summary>
+        /// Totp Issuer.
+        /// </summary>
+        [JsonProperty("totp_issuer")]
+        public string Issuer { get; }
+
+        /// <summary>
+        /// Totp user.
```

Comment:
Could we use the proper casing for the initialism since this is in a non-programmatic context?

```suggestion
        /// TOTP user.
```